### PR TITLE
Cron alpha

### PR DESCRIPTION
**Pull request by rascaraficci**
Added some improvements

- Persistence of jobs in database
- Usage of dojot logging library

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
 Cron is a dojot's microservice that allows you to schedule events to be emitted to other microservices.
 
 This service is under development, and new commits are comming soon.
 
-# TODO
+## TODO
 
-- Persist jobs' configuration.
-- Handle cron's restarts.
+- Change to use dojot's logging library
```
Comment (raulnegreiros) [inline README.md]: Is missing some point? I've seen several uses of dojot's logger during this PR.

File: cron.js [hunk 2]
```diff
@@ -26,101 +29,205 @@ class CronManager {
...
+
+            // db -job
+            let dbEntry = {
+                jobId: jobId,
+                spec: jobSpec
+            };
+            this.db.create(tenant, dbEntry).then(() => {
+                this._setCronJob(tenant, jobId, jobSpec).then(() => {
+                    resolve(jobId);
+                });
```
Comment (raulnegreiros) [inline cron.js:233]: I think is missing a `catch` clause here to treat when the _setCronJob fails.

File: db.js [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    }
+
+    init() {
+        return mongo.connect(config.cronManager.db.mongodb.url, 
+            config.cronManager.db.mongodb.options).then((client) => {
+            this.client = client;
+          });
+    }
+
+    setTenant(tenant) {
```
Comment (raulnegreiros) [inline db.js]: This name does not sound clear in the class context

File: README.md (hunk 1 above)
Comment (rascaraficci) [reply README.md]: In the commit when I added this TODO item, the cron service wasn't using the logging library. But I forgot to remove it when later I changed to use the library.

File: cron.js (hunk 2 above)
Comment (rascaraficci) [reply cron.js:233]: You're right!

File: db.js (hunk 3 above)
Comment (rascaraficci) [reply db.js]: Any suggestion?

Comment (rascaraficci) [reply db.js]: createDatabase(tenant)?

**Review by raulnegreiros — APPROVED** [review]
Great

**Pull request merged**